src/services/tracing.ts

```diff
@@ -244,6 +244,7 @@ const recordException = (span: Span, exception: Exception, level?: ErrorLevel) =
   span.setAttribute("error.level", errorLevel)
   // @ts-ignore-next-line no-implicit-any error
   span.setAttribute("error.name", exception["name"])
+  span.setAttribute("error.message", exception["message"])
```

Comment:
Can you confirm too that `exception["message"]` doesn't break if we don't include a message?